java-manta-client/src/main/java/com/joyent/manta/client/multipart/CreateMPURequestBody.java

```diff
@@ -0,0 +1,115 @@
+package com.joyent.manta.client.multipart;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.joyent.manta.client.MantaMetadata;
+import com.joyent.manta.http.MantaHttpHeaders;
+import org.apache.commons.collections4.map.CaseInsensitiveMap;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpHeaders;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Value object used to represent the JSON payload sent to Manta when creating
+ * a new multipart upload.
+ *
+ * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @since 3.0.0
+ */
+class CreateMPURequestBody {
```

Comment:
The idea was that it is a `RequestBody` for the `CreateMPU` API call. Do you have any ideas on a better name?